lib/internal/v8_prof_polyfill.js

```diff
@@ -154,8 +158,8 @@ function macCppfiltNm(out) {
   }
 
   let i = 0;
-  filtered = filtered.split('\n');
-  return out.replace(FUNC_RE, (all, prefix, postfix) => {
+  filtered = StringPrototypeSplit(filtered, '\n');
+  return StringPrototypeReplace(out, FUNC_RE, (all, prefix, postfix) => {
```

Comment:
Note that calling `StringPrototypeReplace` on a RegExp is unsafe, as it calls `RegExp.prototype[Symbol.replace]`.

---

This should instead use:
```suggestion
  return RegExpPrototypeSymbolReplace(FUNC_RE, out, (all, prefix, postfix) => {
```